app/components/hearings-list-for-milestones-list.js

```diff
@@ -68,13 +68,23 @@ export default class HearingsListForMilestonesListComponent extends Component {
     const dispositions = milestone.get('project.dispositions');
     const milestoneParticipantReviewLookup = this.get('milestoneParticipantReviewLookup');
 
+    const filteredDispositionsArray = [];
+
     // Iterate through ALL of the current project's dispositions.
     // Filter by IF a single disposition's dcpRecommendationsubmittedbyname matches the
     // current milestone's displayName based on the milestoneParticipantReviewLookup.
     // disposition.dcpRecommendationsubmittedbyname = e.g. 'QNBP'
     // disposition.dcpRecommendationsubmittedbyname.substring(2,4) = e.g. 'BP'
     // matching e.g. 'BP' with the milestoneParticipantReviewLookup provides 'Borough President Review'
-    return dispositions.filter(disposition => milestoneParticipantReviewLookup[milestone.displayName] === disposition.dcpRecommendationsubmittedbyname.substring(2, 4));
+    dispositions.forEach(function(disposition) {
+      // make sure that dcpRecommendationsubmittedbyname is not NULL
+      if (disposition.dcpRecommendationsubmittedbyname) {
+        if (milestoneParticipantReviewLookup[milestone.displayName] === disposition.dcpRecommendationsubmittedbyname.substring(2, 4)) {
+          filteredDispositionsArray.push(disposition);
+        }
+      }
+    });
+    return filteredDispositionsArray;
```

Comment:
You can use a .filter then .map but this is fine considering time constraints

Reply:
@allthesignals Can you explain how that would fix the `NULL` error? The error was 'cannot find substring of null` so it would still error if I did that filter first.